<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>action_frame</class>
 <widget class="QWidget" name="action_page">
 <widget class="QFrame" name="action_frame">
  <property name="geometry">
   <rect>
    <x>8</x>
    <y>8</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <widget class="QLabel" name="action_title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1880</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>action title label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="action_buttons_layout_widget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="action_buttons_layout">
    <property name="spacing">
     <number>32</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="action_labels_layout_widget">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>130</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="action_labels_layout">
    <property name="spacing">
     <number>32</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
 </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
